packages/plugins/rubygem-foreman_ansible/rubygem-foreman_ansible.spec

```diff
@@ -1,76 +1,168 @@
+BuildRequires: npm(babel-plugin-lodash) < 4.0.0
+BuildRequires: npm(babel-plugin-transform-class-properties) >= 6.24.1
+BuildRequires: npm(babel-plugin-transform-class-properties) < 7.0.0
+BuildRequires: npm(babel-plugin-transform-object-assign) >= 6.22.0
+BuildRequires: npm(babel-plugin-transform-object-assign) < 7.0.0
+BuildRequires: npm(babel-plugin-transform-object-rest-spread) >= 6.26.0
+BuildRequires: npm(babel-plugin-transform-object-rest-spread) < 7.0.0
+BuildRequires: npm(babel-preset-env) >= 1.6.0
+BuildRequires: npm(babel-preset-env) < 2.0.0
+BuildRequires: npm(babel-preset-react) >= 6.24.1
+BuildRequires: npm(babel-preset-react) < 7.0.0
+#BuildRequires: npm(enzyme) >= 3.7.0
+#BuildRequires: npm(enzyme) < 4.0.0
+#BuildRequires: npm(enzyme-adapter-react-16) >= 1.7.0
+#BuildRequires: npm(enzyme-adapter-react-16) < 2.0.0
+#BuildRequires: npm(enzyme-to-json) >= 3.3.5
+#BuildRequires: npm(enzyme-to-json) < 4.0.0
+#BuildRequires: npm(eslint) >= 4.18.1
+#BuildRequires: npm(eslint) < 5.0.0
+#BuildRequires: npm(eslint-config-airbnb) >= 16.0.0
+#BuildRequires: npm(eslint-config-airbnb) < 17.0.0
+#BuildRequires: npm(eslint-plugin-import) >= 2.8.0
+#BuildRequires: npm(eslint-plugin-import) < 3.0.0
+#BuildRequires: npm(eslint-plugin-jest) >= 21.2.0
+#BuildRequires: npm(eslint-plugin-jest) < 22.0.0
+#BuildRequires: npm(eslint-plugin-jsx-a11y) >= 6.0.2
+#BuildRequires: npm(eslint-plugin-jsx-a11y) < 7.0.0
+#BuildRequires: npm(eslint-plugin-patternfly-react) >= 0.2.1
+#BuildRequires: npm(eslint-plugin-patternfly-react) < 1.0.0
+#BuildRequires: npm(eslint-plugin-react) >= 7.4.0
+#BuildRequires: npm(eslint-plugin-react) < 8.0.0
+BuildRequires: npm(identity-obj-proxy) >= 3.0.0
+BuildRequires: npm(identity-obj-proxy) < 4.0.0
+#BuildRequires: npm(jest) >= 23.6.0
+#BuildRequires: npm(jest) < 24.0.0
+#BuildRequires: npm(prettier) >= 1.16.4
+#BuildRequires: npm(prettier) < 2.0.0
+BuildRequires: npm(react-redux) >= 5.0.7
+BuildRequires: npm(react-redux) < 6.0.0
+#BuildRequires: npm(react-redux-test-utils) >= 0.1.1
+#BuildRequires: npm(react-redux-test-utils) < 1.0.0
+BuildRequires: npm(redux) >= 3.7.2
+BuildRequires: npm(redux) < 4.0.0
+BuildRequires: npm(redux-thunk) >= 2.3.0
+BuildRequires: npm(redux-thunk) < 3.0.0
+# end package.json devDependencies BuildRequires
+
+# start package.json dependencies BuildRequires
+BuildRequires: npm(babel-polyfill) >= 6.26.0
+BuildRequires: npm(babel-polyfill) < 7.0.0
+BuildRequires: npm(classnames) >= 2.2.5
+BuildRequires: npm(classnames) < 3.0.0
+BuildRequires: npm(lodash) >= 4.17.11
+BuildRequires: npm(lodash) < 5.0.0
+BuildRequires: npm(patternfly) >= 3.58.0
+BuildRequires: npm(patternfly) < 4.0.0
+BuildRequires: npm(patternfly-react) >= 2.25.4
+BuildRequires: npm(patternfly-react) < 3.0.0
+BuildRequires: npm(prop-types) >= 15.6.2
+BuildRequires: npm(prop-types) < 16.0.0
+BuildRequires: npm(react) >= 16.6.3
+BuildRequires: npm(react) < 17.0.0
+BuildRequires: npm(react-bootstrap) >= 0.32.4
```

Comment:
Looks like we currently ship 0.32.1. Patternfly-react depends on `< 0.33.0` so we should be able to bump it. This needs an independent packaging PR since it's a build dependency.

Reply:
that would mean bumping also in foreman core, which is probably too late for 1.22, I'll check if we can relax this in foreman_ansible and use 0.32.1 there, seems it will work